addons/api/tests/unit/serializers/role-test.js

```diff
@@ -112,6 +112,7 @@ module('Unit | Serializer | role', function (hooks) {
           authorized_actions: [],
           name: 'Role 1',
           grant_strings: ['*', '*'],
+          grant_scope_ids: [],
```

Comment:
Should we pass in data to test if it correctly serializes, or do we already have one?

Reply:
ah good point, i dont think there's one yet. On it

Reply:
updated!